contacts_vs_distance.ipynb

```diff
@@ -44,7 +44,7 @@
    "outputs": [],
```

Comment:
<p>We explore the influence of sequencing coverage on calculating P(s) curves using <code>cooltools.sample()</code> to generate downsampled coolers from the Kreitenstein microC test dataset. We then compute raw P(s) and smoothed P(s) at various levels of downsampling.</p><p> </p><p>Raw P(s) appear very noisy even with 0.1% downsampling. Smoothed P(s), however, are fairly consistent even after .01% downsampling.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>